doc/start/quick-ceph-deploy.rst

```diff
@@ -117,10 +117,12 @@ configuration details, perform the following steps using ``ceph-deploy``.
 
 	ssh node2
 	sudo mkdir /var/local/osd0
+	sudo chown ceph:ceph /var/local/osd0
```

Comment:
@tahia-khan (NIT) Please consider to use title < 50 characters for patch.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits